.github/workflows/pr-build.yaml

```diff
@@ -1,7 +1,7 @@
 name: PR build
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
This is put just for demo purpose, I will change this back to `pull_request_target` before merging